google-cloud-bigtable/src/test/resources/logging-verbose.properties

```diff
@@ -0,0 +1,17 @@
+handlers= java.util.logging.ConsoleHandler
```

Comment:
please add a comment above that this is used by `enable-verbose-grpc-logs` maven profile